documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -122,9 +122,9 @@ For more information, refer to the xref:type-BrokerCapacity-reference[].
 
 Cruise Control has its own configurable logger:
 
-* `cruisecontrol.root.logger`
+* `rootLogger.level`
 
-Cruise Control uses the Apache `log4j` logger implementation.
+Cruise Control uses the Apache `log4j2` logger implementation.
```

Comment:
```suggestion
Cruise Control uses the Apache `Log4j 2` logger implementation.
```

Reply:
Formatting should be changed to `Log4j 2` across the documentation in a future PR.